docs/Upgrade-Guide/upgrade_to_3.12.md

```diff
@@ -42,18 +43,23 @@ This procedure involves upgrading **one server at a time**, while keeping the vo
     #gluster volume status
 ```
 
-7. Self-heal all gluster volumes by running
+7. Start glustereventsd service (if it was enabled before), either using the command below, or through other means,
```

Comment:
If the glustereventsd service was previously enabled, it is required to start it using the commands below, or, through other means

Reply:
Done